budgetApp1/budgetApp1/AccountViewController.swift

```diff
@@ -104,9 +74,8 @@ extension AccountViewController: UITableViewDelegate, UITableViewDataSource {
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         if segue.identifier == Segues.accountTransactionsSegue,
             let selectedIndexPath = tableView.indexPathForSelectedRow,
-            let navigationController = segue.destination as? UINavigationController,
-            let accountTransactionsVC = navigationController.topViewController as? AccountTransactionsViewController {
-            let selectedAccount = viewModel.account(atIndex: selectedIndexPath.section)
+           let accountTransactionsVC = segue.destination as? AccountTransactionsViewController {
```

Comment:
indenting looks off. Please select all code in all files and press control + I to indent it all

Reply:
indenting still off. Please push your changes and only AFTER that then mark the conversation as resolved..